src/plugins/intel_npu/src/backend/src/zero_init.cpp

```diff
@@ -201,8 +200,44 @@ ZeroInitStructsHolder::ZeroInitStructsHolder() : log("NPUZeroInitStructsHolder",
     // Get our target device
     zeroUtils::throwOnFail("zeDeviceGet", zeDeviceGet(driver_handle, &device_count, &device_handle));
 
+    // Create context from backend to be shared with compiler
     ze_context_desc_t context_desc = {ZE_STRUCTURE_TYPE_CONTEXT_DESC, 0, 0};
     zeroUtils::throwOnFail("zeContextCreate", zeContextCreate(driver_handle, &context_desc, &context));
+
+    // Allocate memory for the char array
+    graphExtName = new char[graph_ext_name.size() + 1];
+    // Copy the contents of the string to the char array
+    std::strcpy(graphExtName, graph_ext_name.c_str());
+
+    const uint16_t adapterMajorVersion = 1;
```

Comment:
if this is really needed could we make it global?

Reply:
i think this check might not be useful, so i remove this part in the latest commit